docs/patterns/single-new-eks-observability-accelerators/single-new-eks-opensource-observability-graviton.md

```diff
@@ -0,0 +1,331 @@
+# Singe New EKS Cluster Open Source Observability Accelerator
```

Comment:
Fix the heading. It is missing Graviton. Also i wanted it be called as `single-eks-graviton-opensource-observability` everywhere, docs, file naming, code etc etc.